airbyte-integrations/connectors/source-freshdesk/integration_tests/expected_records.jsonl

```diff
@@ -1,132 +1,47 @@
-{"stream": "conversations", "data": {"body": "<div>test</div>", "body_text": "test", "id": 67105779604, "incoming": true, "private": false, "user_id": 67042965499, "support_email": null, "source": 6, "category": 7, "to_emails": null, "from_email": null, "cc_emails": [], "bcc_emails": null, "email_failure_count": null, "outgoing_failures": null, "thread_id": null, "thread_message_id": null, "created_at": "2022-11-22T15:02:29Z", "updated_at": "2022-11-22T15:02:29Z", "last_edited_at": null, "last_edited_user_id": null, "attachments": [], "automation_id": null, "automation_type_id": null, "auto_response": false, "ticket_id": 23, "source_additional_info": null}, "emitted_at": 1672878361522}
-{"stream": "satisfaction_ratings", "data": {"id": 67000499229, "survey_id": 67000084490, "user_id": 67042965499, "agent_id": 67013469919, "feedback": "test", "group_id": null, "ticket_id": 23, "created_at": "2022-11-22T15:02:29Z", "updated_at": "2022-11-22T15:02:29Z", "ratings": {"default_question": 103, "question_67000084816": 103, "question_67000084817": 100, "question_67000084818": 100}}, "emitted_at": 1672878362116}
-{"stream": "surveys", "data": {"id": 67000039187, "title": "Default Survey", "active": false, "created_at": "2020-10-22T02:37:14Z", "updated_at": "2022-11-16T16:24:54Z", "questions": [{"id": "default_question", "label": "How would you rate your overall satisfaction for the resolution provided by the agent?", "accepted_ratings": [103, 100, -103], "default": true}]}, "emitted_at": 1672878362646}
-{"stream": "surveys", "data": {"id": 67000084490, "title": "Test survey", "active": true, "created_at": "2022-11-16T16:24:00Z", "updated_at": "2022-11-22T15:00:30Z", "questions": [{"id": "default_question", "label": " How would you rate your overall satisfaction for the resolution provided by the agent? ", "accepted_ratings": [103, 100, -103], "default": true}, {"id": "question_67000084816", "label": "Question 1?", "accepted_ratings": [103, 100, -103]}, {"id": "question_67000084817", "label": "Question 2?", "accepted_ratings": [103, 100, -103]}, {"id": "question_67000084818", "label": "Are you satisfied with our customer support experience?", "accepted_ratings": [103, 100, -103]}]}, "emitted_at": 1672878362647}
+{"stream": "agents", "data": {"available": false, "occasional": true, "id": 67021120885, "ticket_scope": 1, "created_at": "2021-02-11T22:21:43Z", "updated_at": "2021-12-01T00:09:08Z", "last_active_at": null, "available_since": null, "type": "support_agent", "contact": {"active": false, "email": "test_agent_8@test.com", "job_title": null, "language": "en", "last_login_at": null, "mobile": null, "name": "Test Agent 8", "phone": null, "time_zone": "Magadan", "created_at": "2021-02-11T22:21:43Z", "updated_at": "2021-02-11T22:21:43Z"}, "signature": "<div dir=\"ltr\"><p><br></p>\n</div>"}, "emitted_at": 1675116877912}
```

Comment:
It looks like these expected records now have `signature`s that are not null - what are these? Asking because I don't see it in the changelog

Reply:
Some deleted `agents` streams are also have `signatures`. I don't understand what was the question